package.json

```diff
@@ -4,12 +4,13 @@
   "description": "An adapter for connecting Open MCT with YAMCS",
   "main": "dist/openmct-yamcs.js",
   "scripts": {
-    "clean": "rm -rf ./dist ./node_modules ./package-lock.json",
+    "clean": "rm -rf ./dist ./node_modules ./package-lock.json ./coverage ./tests/html-test-results ./tests/e2e/test-results ./.nyc_output ./tests/e2e/opensource",
     "test": "npm run test:getopensource && npm run test:e2e:smoke",
     "lint": "eslint src example",
     "lint:fix": "eslint src example --fix",
     "build:dist": "webpack --config webpack.prod.js",
     "build:example": "npm install openmct@unstable --no-save",
+    "build:example:master": "npm install nasa/openmct#master --no-save",
```

Comment:
Nice, this would have been my approach as well. That way the "example" is always running with stable openmct.